### PR TITLE
Added excerpt helper

### PR DESCRIPTION
**Pull request by mattmezza**
I've added an `{{excerpt post_body 20 '[Read more...]'}}` helper for truncating without cutting words.

Thanks for the awesome set of helpers.

### CODE REVIEW

**Comment by JustBlackBird** [comment]
Hi!

First of all I want to thank you for the PR. I have one big question about it: why don't you name the helper just `ellispsis`?

File: src/Text/ExcerptHelper.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+* Excerpt
+* (c) Matteo Merola
```
Comment (JustBlackBird) [inline src/Text/ExcerptHelper.php]: This license is for the library, not for the single file, so you have to use the same header as used in other files.

Comment (mattmezza) [reply src/Text/ExcerptHelper.php]: Got it.

**Comment by mattmezza** [comment]
Hello, 
I don't know about the name actually. I've used `excerpt` because usually this kind of feature is used in the place of truncate when I want to avoid cutting text with unfinished words (that is useful especially in long texts e.g. blog posts etc...). Therefore I used `excerpt`, but probably a longer `truncate_with_words` or `truncate_words` might be more self explanatory.

Let me know what you think and I'll make the changes so that you can release the new version.

**Comment by JustBlackBird** [comment]
As for me, `excerpt` is not an obvious name for the helper. I'd name it just `ellipsis`, because such name is often used in other libraries for the same purpose.

In contrasts `truncate` is just cut the string wherever it said to. Also we already have `truncate` helper and one more helper with `truncate*` prefix could be misleading.

**Comment by mattmezza** [comment]
Ok, you got me 👍. I'll make the changes and update this PR with the new commit for you to merge. Maybe tomorrow or the following days.

**Comment by JustBlackBird** [comment]
I've thought for a while about the helper's name, so what do you think about `teaser`?

**Comment by mattmezza** [comment]
I don't know, I'd stick with ellipsis if I'd have to choose between them, lemme know as I plan to make the changes during the next couple of hours 👍

**Comment by JustBlackBird** [comment]
Actually it's you, who choose the name for the helper. I've tell you about `teaser` because I thought that you could like it more than `ellipsis`. If you don't, just stick with `ellipsis`, it's fine.

**Comment by mattmezza** [comment]
Perfect then, thank you very much. It is always something good to discuss about important things such as names. I hope you don't have to wait that long for the PR. 

Thank you again.

**Comment by mattmezza** [comment]
I applied the changes. Sorry for the delay.

Have a nice day.

**Comment by JustBlackBird** [comment]
Thank your for the PR! Your changes were released in [v1.2.0](https://github.com/JustBlackBird/handlebars.php-helpers/releases/tag/v1.2.0)

**Pull request merged**